.github/workflows/ci.yml

```diff
@@ -98,24 +98,34 @@ jobs:
     - run: yarn test-proto # Run before unit-core because the roundtrip json is needed for proto tests.
 
     - run: sudo apt-get install xvfb
+      if: matrix.os == 'ubuntu-latest'
+
     - name: yarn unit
       run: xvfb-run --auto-servernum yarn unit
+      if: matrix.os == 'ubuntu-latest'
+    - name: yarn unit
```

Comment:
the `unit` test stuff is almost entirely pure-js (xvfb is only needed for `unit-cli`, I think?), so it might be a better use of testing time to do more potentially platform-dependent stuff like `test-clients`/`test-viewer`/`diff:sample-json`?

Reply:
As a counteropinion, I care about `unit-cli` on windows a lot more than I care about `diff:sample-json` :)

isn't diff:sample-json like 100% JS in auditing phase and fixed artifacts without any chrome launching/gathering involved and weird os-specific CLI pathing quirks? I guess there's the bash invocation of LH in there which works, but still smokes+unit-cli are like *all* I want to see on windows 😄

Reply:
> As a counteropinion, I care about `unit-cli` on windows a lot more than I care about `diff:sample-json` :)

I'm not against any particular test, I'm just thinking if we want to spend ~5m testing, what's the best bang for its buck. `yarn unit-cli` seems like it could be a good use of it ([27s for `unit-cli`](https://github.com/GoogleChrome/lighthouse/pull/11087/checks?check_run_id=871877126#step:11:279) vs 6m23s for all of `unit`).

For `diff:sample-json`, it also runs `collect-strings`, and that through `update:sample-json` is a key part of our development pipeline. And it takes 5s :)

Reply:
how's that?

Reply:
wfm

Reply:
I was thinking this could also just live in the windows test rather than split `unit` up